recipe/build.sh

```diff
@@ -84,8 +84,8 @@ if [[ "$host_platform" != "$build_platform" ]]; then
        $extra_host_options
 
     mkdir -p gcc/include-fixed
-    cp ../gcc/gcc-ar.c gcc/gcc-nm.c
-    cp ../gcc/gcc-ar.c gcc/gcc-ranlib.c
+    cp ../gcc/gcc-ar.cc gcc/gcc-nm.cc || cp ../gcc/gcc-ar.c gcc/gcc-nm.c
+    cp ../gcc/gcc-ar.cc gcc/gcc-ranlib.cc || cp ../gcc/gcc-ar.c gcc/gcc-ranlib.c
```

Comment:
Please don't use my commits without giving me credit.

Reply:
I'm very sorry about that oversight.  😞